packages/node_modules/@ciscospark/react-component-activity-text/src/index.js

```diff
@@ -38,19 +43,22 @@ export default function ActivityText(props) {
       h5: [],
       h6: []
     }, [], content);
-    const htmlContent = {__html: cleaned};
+    const linkifiedContent = linkifyHtml(cleaned, linkifyOptions);
```

Comment:
The `linkifyHtml` is aware of the html within and shouldn't change it. Just tested and it works, but I will add a test suite to make sure.